qa/vagrant/src/test/resources/packaging/scripts/70_sysv_initd.bats

```diff
@@ -117,6 +117,23 @@ setup() {
     [ "$status" -eq 3 ] || [ "$status" -eq 4 ]
 }
 
+@test "[INIT.D] don't mkdir when it contains a comma" {
+    # Remove these just in case they exist beforehand
+    rm -rf /tmp/aoeu,/tmp/asdf
+    rm -rf /tmp/aoeu,
+    # set DATA_DIR to DATA_DIR=/tmp/aoeu,/tmp/asdf
+    sed -i 's/DATA_DIR=.*/DATA_DIR=\/tmp\/aoeu,\/tmp\/asdf/' /etc/init.d/elasticsearch
+    cat /etc/init.d/elasticsearch | grep "DATA_DIR"
+    service elasticsearch start
+    wait_for_elasticsearch_status
+    assert_file_not_exist /tmp/aoeu,/tmp/asdf
```

Comment:
Here's the actual kernel calls for it:

```
...
mkdir("/tmp", 0777)                     = -1 EEXIST (File exists)
chdir("/tmp")                           = 0
mkdir("aoeu,", 0777)                    = 0
open("aoeu,", O_RDONLY|O_NOCTTY|O_NONBLOCK|O_DIRECTORY|O_NOFOLLOW) = 3
fchdir(3)                               = 0
close(3)                                = 0
mkdir("tmp", 0777)                      = 0
open("tmp", O_RDONLY|O_NOCTTY|O_NONBLOCK|O_DIRECTORY|O_NOFOLLOW) = 3
fchdir(3)                               = 0
close(3)                                = 0
mkdir("asdf", 0777)                     = 0
...
```

Full output: https://gist.github.com/dakrone/38f6c4c81995c56de148f48e64e9b952

So it's `/tmp -> aoeu,` and then `tmp -> asdf` which ends up with a tree like:

```
$ tree /tmp/aoeu,/
/tmp/aoeu,/
└── tmp
    └── asdf

2 directories, 0 files
```